src/engines/html5/html5.js

```diff
@@ -1165,7 +1165,7 @@ export default class Html5 extends FakeEventTarget implements IEngine {
     const listenToCueChange = track => {
       track.mode = PKTextTrack.MODE.HIDDEN;
       this._eventManager.listen(track, 'cuechange', () => {
-        this.dispatchEvent(new FakeEvent(CustomEventType.TIMED_METADATA, {cues: Array.from(track.activeCues)}));
+        this.dispatchEvent(new FakeEvent(CustomEventType.TIMED_METADATA, {cues: Array.from(track.activeCues), label: track.label}));
```

Comment:
Label is the data source attribute of the text track but who said times metadata data source will always be a text track.
This is an example of leaking implementation detail.
I assume this was done for "easier" filtering but it leads to tightly coupling text track and the timed metadata event and needs to be removed.